### PR TITLE
Nt add status to ok response

### PR DESCRIPTION
**Pull request by nooralnabi**

### CODE REVIEW

File: app/controllers/api/v1/addresses_controller.rb [hunk 1]
```diff
@@ -6,9 +6,16 @@ def create
...
     authorize address
     if address.save
-      render json: {status: 'Customer address created.'}.merge(main_info: main_info), status: :ok
+      render json: {status: true, message:'Customer address created.'}.merge(main_info: main_info), status: :ok
     else
-      render json: {error: address.errors.full_messages.to_sentence}.merge(main_info: main_info), status: :unprocessable_entity
+      errors_messages=[]
+      address.errors.each do |attr, messages|
+        errors_messages << {
+                            field_name:attr,
```
[the remark this answers is not in the document]
Comment (nooralnabi) [reply app/controllers/api/v1/addresses_controller.rb]: ok

File: app/controllers/api/v1/addresses_controller.rb [hunk 2]
```diff
@@ -17,9 +24,15 @@ def update
     raise 'Record Not Found' unless address
     authorize address
     if address.update_attributes(address_params)
-      render json: {status: 'Address was successfully updated.'}.merge(main_info: main_info), status: :ok
+      render json: {status: true, message: 'Address was successfully updated.'}.merge(main_info: main_info), status: :ok
     else
-      render json: {error: @address.errors.full_messages.to_sentence}.merge(main_info: main_info), status: :unprocessable_entity
+      errors_messages=[]
+      address.errors.each do |attr, messages|
+        errors_messages << {
```
Comment (ssharba) [inline app/controllers/api/v1/addresses_controller.rb]: make it one line everywhere please

**Comment by nooralnabi** [comment]
@MwMaki

**Comment by ssharba** [comment]
@nooralnabi conflicts

**Pull request merged**